android/src/main/kotlin/org/dasfoo/flutter_sentry/FlutterSentryPlugin.kt

```diff
@@ -79,4 +82,14 @@ class FlutterSentryPlugin : FlutterPlugin, MethodCallHandler {
     override fun onDetachedFromEngine(@NonNull binding: FlutterPlugin.FlutterPluginBinding) {
         channel.setMethodCallHandler(null)
     }
+
+    private fun setEnvironment(@NonNull call: MethodCall, @NonNull result: Result) {
+        val environment = call.argument<String>("environment") ?:
+            return result.error("MISSING_PARAMS", "Missing 'environment' parameter", null)
```

Comment:
Will passing `null` to `setEnvironment` in Dart code trigger the "missing parameter" error in native code?

Reply:
This will result in a `PlatformException` being thrown at the dart layer from the `methodChannel.invokeMethod` invocation.

Reply:
Right, I was thinking we can save on a round-trip to native code since we know beforehand that it's going to fail.

Reply:
Makes sense. Will update to do that when I have a moment.